SUMMARY.md

```diff
@@ -142,17 +142,19 @@
       * [Adding permissions](guides/plugins/plugins/administration/add-acl-rules.md)
       * [Add custom route](guides/plugins/plugins/administration/add-custom-route.md)
       * [Writing templates](guides/plugins/plugins/administration/writing-templates.md)
-      * [Add menu module](guides/plugins/plugins/administration/add-custom-module.md)
+      * [Add custom module](guides/plugins/plugins/administration/add-custom-module.md)
       * [Add menu entry](guides/plugins/plugins/administration/add-menu-entry.md)
       * [Add tab to existing module](guides/plugins/plugins/administration/add-new-tab.md)
       * [Customizing components](guides/plugins/plugins/administration/customizing-components.md)
-      * [Add own component](guides/plugins/plugins/administration/add-custom-component.md)
+      * [Add custom component](guides/plugins/plugins/administration/add-custom-component.md)
+      * [Add custom input field to existing component](guides/plugins/plugins/administration/add-custom-field.md)
       * [Adding snippets](guides/plugins/plugins/administration/adding-snippets.md)
       * [Add custom styles](guides/plugins/plugins/administration/add-custom-styles.md)
       * [Using the data handling](guides/plugins/plugins/administration/using-data-handling.md)
       * [Using the data grid component](guides/plugins/plugins/administration/using-the-data-grid-component.md)
       * [Add filter](guides/plugins/plugins/administration/add-filter.md)
       * [Using filter](guides/plugins/plugins/administration/using-filter.md)
+      * [Using the Shopware object](guides/plugins/plugins/administration/the-shopware-object.md)
```

Comment:
Niklas already added that one

Reply:
Are you sure that was complete? I couldn't find the reference in this file and in Gitbook at all ... 🤔

Reply:
https://github.com/shopware/docs/pull/149/files

Reply:
He's adding it in that PR